src/main/java/org/threeten/extra/YearWeek.java

```diff
@@ -473,6 +473,60 @@ public boolean is53WeekYear() {
     public int lengthOfYear() {
         return (is53WeekYear() ? 371 : 364);
     }
+    
+    //-----------------------------------------------------------------------
+    /**
+     * Returns a copy of this year-week with the specified number of weeks added.
+     * * <p>
+     * This instance is immutable and unaffected by this method call.
+     * 
+     * @param weeksToAdd  the weeks to add, may be negative
+     * @return the year-week with the weeks added, not null
+     */
+    public YearWeek plusWeeks(long weeksToAdd) {
+        if (weeksToAdd == 0) {
+            return this;
+        }
+        if (weeksToAdd < 0) {
+            return minusWeeks(-weeksToAdd);
+        }
+        YearWeek yearWeek = this;
+        while (weeksToAdd-- > 0) {
```

Comment:
:winces:
Okay, granted, for small values this probably works.  But for any significant range this is going to be a drag.  
The behavior of the ISO week is well defined, and you should be able to calculate it out - `PaxCalendar` and `AccountingCalendar` likely have something to get you started.  For that matter, switching to a regular date (as if this represented the start of the given week) and back would likely yield reasonable results.